app/Http/Controllers/PageController.php

```diff
@@ -45,6 +45,7 @@ public function create()
     {
         $pages = Page::All();
         $images = Image::All();
+        $components = Component::All();
```

Comment:
Remove this line. We only want parent components, not all of them

Reply:
Removed 👍